### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gaurav-dalvi**
Added 2 steps in prerequisite section.

### CODE REVIEW

File: net/README.md [hunk 1]
```diff
@@ -3,20 +3,24 @@
 ### Pre-requisites
 1. You need Ubuntu 15 or Centos 7 on each of your servers used for the Contiv cluster
 
-2. Do "export CLUSTER_NODE_IPS=ips of all nodes in cluster", and "export no_proxy=ips of all nodes in cluster,127.0.0.1,localhost,netmaster" in your shell
+2. Make sure SELinux is disabled on your nodes in cluster. Use 'set enforce 0' command to do this. For more information please take a look [here](https://www.centos.org/docs/5/html/5.2/Deployment_Guide/sec-sel-enable-disable-enforcement.html)
 
-3. If your servers are behind an http proxy (usually the case in many cisco labs...), you need
+3. Make sure you are running ansible 2.0.x version on these nodes.
```
Comment (mapuri) [inline net/README.md:8]: Gaurav, I think the installer should take care of this as it already tries to install ansible if it is not installed.

**Comment by gaurav-dalvi** [comment]
Thanks  Madhav. I was running Centos 7 which came with Ansible 1.x, So I bumped up the version of Ansible. So I feel its appropriate to write this step here. If not in installer script should take care of displaying this message or bail from the installation with some error message.

**Comment by mapuri** [comment]
@gaurav-dalvi 

> So I feel its appropriate to write this step here. 

may be I wasn't clear, what I meant was that since installer takes care of installing it it shall take care of updating it and we should have a need to document this as a pre-requisite.

**Comment by gaurav-dalvi** [comment]
Hey Madhav,

You are right. I have other PR with the fix. (via EPEL testing repo, installing ansible 2.0)
Closing this PR now as Its not required.

**Pull request closed without merging**